src/Paulus/Exception/Http/Standard/HttpExceptionFactory.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Paulus - A PHP micro-framework for creating RESTful services
+ *
+ * @author      Trevor Suarez (Rican7)
+ * @author      Mark Costello (mcos)
+ * @copyright   2015 Trevor Suarez
+ * @link        https://github.com/Rican7/Paulus
+ * @license     https://github.com/Rican7/Paulus/blob/master/LICENSE
+ * @version     2.0.0
+ */
+
+namespace Paulus\Exception\Http\Standard;
+
+use Exception;
+
+/**
+ * HttpExceptionFactory
+ *
+ * Provides the means to create Api Exceptions using minimal other information.
+ *
+ * @uses Exception
```

Comment:
This is gonna be annoying, especially since the doc-block tags in this repo could use a serious update, but this tag is traditionally used in this repo for classes that extend or implement another. So you can remove this.